tests/console/yast2_registration.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: check if an unregistered system can be registered and if
+#          enabling and disabling extensions correctly work.
+# Maintainer: Paolo Stivanin <pstivanin@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use utils 'zypper_call';
+
+sub register_system_and_add_extension {
+    wait_screen_change { type_string get_var "SCC_EMAIL" };
+    send_key "tab";
+    wait_screen_change { type_string get_var "SCC_REGCODE" };
+    send_key "alt-n";
+    assert_screen("yast2_registration-ext-mod-selection", timeout => 60);
+    # enable Web and Scripting Module
+    for my $i (0 .. 25) {
+        send_key "down";
+    }
+    send_key "spc";
+    send_key "alt-n";
+    wait_still_screen(stilltime => 7, timeout => 60);
+    if (check_screen "yast2_registration-license-agreement") {
+        send_key "alt-a";
```

Comment:
two send_key operations without wait or assert_screen can lead to unstable test

Reply:
sorry, notification got lost in the email. Working on it right now :+1: